ZelBack/config/default.js

```diff
@@ -1,6 +1,6 @@
-const userconfig = require('../../config/userconfig');
+let userconfig = require('../../config/userconfig'); // eslint-disable-line
```

Comment:
why is this let?

Reply:
or it need to be const but we dont want it

Reply:
I seem to be able to operate, build ui without it with following:

const userconfig = require('../../config/userconfig');

const isDevelopment = userconfig.initial.development || false;

Reply:
I pushed the correction to this. Can't figure out why let or eslint disable is needed